Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md

```diff
@@ -508,6 +508,28 @@ case .startButtonTapped:
 This greatly reduces the bandwidth of actions being sent into the system so that you are not 
 incurring unnecessary costs for sending actions.
 
+Another example that comes up often is sliders. If done in the most direct way, by deriving a 
```

Comment:
Unrelated to the core of this PR, but this came up in slack today and thought it would be good to add. I think the example before this of limiting actions in effects is a little complex, and so a simpler example of a `Slider` would be nice.